storage/src/vespa/storage/bucketdb/striped_btree_lockable_map.hpp

```diff
@@ -0,0 +1,315 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+#pragma once
+
+#include "striped_btree_lockable_map.h"
+#include "btree_lockable_map.hpp"
+#include <algorithm>
+#include <cassert>
+#include <queue>
+
+namespace storage::bucketdb {
+
+namespace {
+
+constexpr uint8_t used_bits_of(uint64_t key) noexcept {
+    return static_cast<uint8_t>(key & 0b11'1111ULL);
+}
+
+}
+
+// TODO rename to sharded_btree_lockable_map instead?
```

Comment:
I think we can remove this.

Reply:
Already removed in a subsequent commit 🙂